### PR TITLE
fix: Add check on language files and add missing translations for en and es files

### PR DESCRIPTION
**Pull request by Ldoppea**
Some languages files were missing translations and also extra languages were not in sync with each other in white labels

Those new checks should ensure that all langages files contain the same keys and that all extra language override the same keys

### CODE REVIEW

**Comment by Ldoppea** [comment]
@KillianCourvoisier can you review the new Spanish translations?

File: white_label/configure-brand.ts [hunk 1]
```diff
@@ -25,6 +26,20 @@ export const checkCozyBrandIso = async (): Promise<boolean> => {
...
 }
 
+export const checkLanguages = (): boolean => {
+  const areEnAndFrEquivalent = areLanguagesEquivalent('en', 'es')
+  const areEnAndEsEquivalent = areLanguagesEquivalent('en', 'fr')
+  const areLangagesOk = areEnAndFrEquivalent && areEnAndEsEquivalent
+
+  const areExtraFrEquivalent = areExtraLanguagesEquivalent('en')
+  const areExtraEnEquivalent = areExtraLanguagesEquivalent('en')
+  const areExtraEsEquivalent = areExtraLanguagesEquivalent('fr')
```
Comment (zatteo) [inline white_label/configure-brand.ts:36]: `en en fr` where as it should be `fr en es`? Or I did not understand?

Comment (zatteo) [reply white_label/configure-brand.ts:36]:
```
const extraLanguages = ['fr', 'en', 'es']
const areExtraLangagesOk = extraLanguages.every(extraLanguage => areExtraLanguagesEquivalent(extraLanguage))
```

seems clearer ?

Comment (Ldoppea) [reply white_label/configure-brand.ts]: Nice catch, i fixed it [here](https://github.com/cozy/cozy-flagship-app/compare/1bd78f72075b6bad4eb44961893d75f51de6a096..2bd5d7d0f4158d8ca4c3dc10ef1ca8a2e62e5b38)

Some considerations:
- I applied your suggestion also for `areLanguagesEquivalent` calls
- This implies that there will be a useless call to `areLanguagesEquivalent('en', 'en')`, comparing english to itself
  - I consider this not a problem as the result will be always be `true` and this is fast enough to not be perceived. Is is acceptable considering the improvement in code readibility
- I could have put `areLanguagesEquivalent` and `areExtraLanguagesEquivalent` in the same loop, but I prefer it this way so the console prints will be grouped by file type (languages, then extra-languages)

**Review by zatteo — APPROVED** [review]

**Review by KillianCourvoisier — APPROVED** [review]
LGTM 👍

**Pull request merged**